src/cmd/node/rpc/mod.rs

```diff
@@ -12,7 +12,7 @@ use std::fmt;
 use std::path::Path;
 
 /// Reads bytes from a file for the given path.
-fn read_bytes(file_path: &Path) -> Result<Vec<u8>> {
+pub(crate) fn read_bytes(file_path: &Path) -> Result<Vec<u8>> {
```

Comment:
We could move this `fn` somewhere else eventually, maybe into `src/mod.rs`

Reply:
Done